internal/system_metrics.pb.go

```diff
@@ -208,7 +207,8 @@ func (m *SystemMetricsArtifactInfo) GetSchema() SchemaType {
 
 type SystemMetricsData struct {
 	Id                   string   `protobuf:"bytes,1,opt,name=id,proto3" json:"id,omitempty"`
-	Data                 []byte   `protobuf:"bytes,2,opt,name=data,proto3" json:"data,omitempty"`
+	MetricType           string   `protobuf:"bytes,2,opt,name=metricType,proto3" json:"metricType,omitempty"`
```

Comment:
Can this just be type?

Reply:
type is a reserved word, so it would be awkward for private fields in structs

Reply:
Sure, but this is an exported field so you can use `Type` but not `type` as a field name.